Library/Formula/svmlight.rb

```diff
@@ -0,0 +1,16 @@
+class Svmlight < Formula
+  homepage "http://svmlight.joachims.org/"
+  url "http://download.joachims.org/svm_light/current/svm_light.tar.gz"
```

Comment:
I see.  l just sent the author an email.  Is there a particular convention that should be followed?